parcellab/common/templates/_container.tpl

```diff
@@ -0,0 +1,82 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+  Keys and quoted values generated from a given dict:
+  {{ include "common.container" (
+    dict
+      "container" "The specific pod configuration"
+      "type" "The tye of container to define /optional (defaults to 'service')"
+  ) }}
+*/}}
+{{- define "common.container" -}}
+{{- $commonRefName := .commonRefName -}}
+{{- $name := .name -}}
+{{- $image := .image -}}
+{{- $containerEnv := .containerEnv -}}
+{{- $podVolumes := .volumes -}}
+{{- $podSecurityContext := .podSecurityContext -}}
+{{- $datadog := .datadog -}}
+{{- $commonExternalSecret := .commonExternalSecret -}}
+{{- $commonConfig := .commonConfig -}}
+- name: {{ $name }}
```

Comment:
I would not put the list here. Then you can put it outside and it gives you the freedom to define a single container. The list with `-` limits you in that regard